spec/lib/api/2.0/tasks-spec.js

```diff
@@ -37,8 +37,26 @@ describe('Http.Api.Tasks', function () {
           });
     });
 
+    afterEach('restoring stubs', function() {
+        function resetStubs(obj) {
+            _(obj).methods().forEach(function (method) {
+                if (obj[method] && obj[method].reset) {
+                    obj[method].reset();
+                }
+            }).value();
+        }
+
+        resetStubs(taskProtocol.activeTaskExists);
+        resetStubs(taskProtocol.requestCommands);
+        resetStubs(taskProtocol.respondCommands);
+        resetStubs(tasksApiService.getNode);
+        resetStubs(lookupService.ipAddressToMacAddress);
+    });
+
     after('stop HTTP server', function () {
-        return helper.stopServer();
+        return helper.reset().then(function(){
```

Comment:
why does this need call the `helper.reset()` but other unit-test files don't need?